src/use-internet-time.ts

```diff
@@ -3,22 +3,26 @@ import { now } from "dot-beat-time";
 
 type InternetTimeHookOptions = {
   fractional?: boolean;
+  ssr?: boolean;
 };
 
 function useInternetTime(options?: InternetTimeHookOptions) {
   const isFractional = options && options.fractional;
+  const ssr = options && options.ssr;
 
-  const [time, setTime] = useState(now(isFractional));
```

Comment:
I don't know much about SSR — why would a mismatch of "221.50" and "221.51" cause a problem, and not "" and "221.51"?

Reply:
"" and "221.51" would be a problem, for sure. That's the issue we would encounter if this line looked like this (which was my first attempt):

```ts
const [time, setTime] = useState(typeof window === "undefined" ? "" : now(isFractional));
```

That's why I opted for the `ssr` config option, which will be true on both the server _and_ the client. With this approach, as long as a developer specifies `ssr: true`, this state initializer will result in matching values of "" and "" in both environments.